rpm/felix.spec

```diff
@@ -210,6 +210,28 @@ rm -rf $RPM_BUILD_ROOT
     - Rev libcalico-go to pick up resync loop fix.
     - DatastoreType etcdv2 -> etcdv3
 
+* Mon Oct 16 2017 Neil Jerram <neil@tigera.io> 2.7.0-0.1.pre2
+  - Felix 2.7.0-pre2 (from Git commit 942db10).
+    [Changes recorded in 2.7.0-pre2 tag]
+    - Add support for network sets.
+
+* Mon Oct 16 2017 Neil Jerram <neil@tigera.io> 2.7.0-0.1.pre1
+  - Felix 2.7.0-pre1 (from Git commit d0576c7).
+    [Changes recorded in 2.7.0-pre1 tag]
+    - Implement support for network sets.
+    [Changes recorded in 2.6.0 tag]
```

Comment:
As with debian/changelog, remove the 2.6.0 material here.

Reply:
I might just remove all the 2.7-pre release stanzas, they were pulled in by the merge and I'm not sure they're that useful.

Reply:
Ok, if you prefer.  I thought you were intentionally wanting to record the 2.7.0-pre1 delivery, and that that was quite a nice touch.